### ISSUE TITLE
Texturepack controller

### ISSUE THREAD

**Issue by Thunder33345** [ImpactDevelopment/ImpactClient#593]
Allow users to control the texturepack
for example, moving local above server packs 
for example, reject server packs without server knowing

**Comment by Notoh** [comment]
I have a method from another project that allows for this. I can pr it if interested

**Comment by LeafHacker** [comment]
@Notoh I see no reason why not, even if only to act as inspiration

**Comment by ZeroMemes** [comment]
No-SRP (What is described in #118 and by Notoh) has been added in 4.4. Will look into adjusting local packs over server packs.
